plugin.video.manoto/default.py

```diff
@@ -1,115 +1,10 @@
-import urllib,urllib2,re,os,cookielib,string
-import xbmcplugin,xbmcgui,xbmcaddon
-from BeautifulSoup import BeautifulSoup
-
-addon = xbmcaddon.Addon('plugin.video.manoto')
-profile = xbmc.translatePath(addon.getAddonInfo('profile'))
-
-
-__settings__ = xbmcaddon.Addon(id='plugin.video.manoto')
-__language__ = __settings__.getLocalizedString
+import xbmc, xbmcgui, os, xbmcaddon
 
+__settings__ = xbmcaddon.Addon()
 home = __settings__.getAddonInfo('path')
 icon = xbmc.translatePath(os.path.join(home, 'icon.png'))
+li = xbmcgui.ListItem ('manoto')
+li.setArt(icon)
```

Comment:
setArt receives a dictionary of values, so it has to be something like `li.setArt({'icon':icon})` . Check the documentation for all the options setArt provides.